src/RoslynInsertionTool/scripts/HelperFunctions.ps1

```diff
@@ -29,6 +29,14 @@ function GetComponentProjectName([string] $componentProjectName) {
     }
 }
 
+function GetComponentUserName([string] $componentAzdoUri) {
+    if (IsDefaultValue $componentAzdoUri) {
+        return ""
+    } else {
+        return "/cbu=vslsnap@microsoft.com"
```

Comment:
The scripts pass the user hardcoded already but the component user needs to be optionally included.